platform.md

```diff
@@ -276,10 +275,72 @@ All lifecycle phases:
+  [-tag <tag>...] \
+  [-uid <uid>] \
+  <image>
+```
+
+##### Inputs
+| Input             | Environment Variable  | Default Value            | Description
+|-------------------|-----------------------|--------------------------|----------------------
+| `<analyzed>`      | `CNB_ANALYZED_PATH`   | `<layers>/analyzed.toml` | Path to output analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+| `<daemon>`        | `CNB_USE_DAEMON`      | `false`                  | Analyze image from docker daemon
+| `<gid>`           | `CNB_GROUP_ID`        |                          | Primary GID of the stack `User`
+| `<group>`         | `CNB_GROUP_PATH`      | `<layers>/group.toml`    | Path to group definition (see [`group.toml`](#grouptoml-toml))
+| `<layers>`        | `CNB_LAYERS_DIR`      | `/layers`                | Path to layers directory
+| `<log-level>`     | `CNB_LOG_LEVEL`       | `info`                   | Log Level
+| `<previous-image>`| `CNB_PREVIOUS_IMAGE`  | `<image>`                | Image reference to be analyzed (usually the result of the previous build)
+| `<run-image>`     | `CNB_RUN_IMAGE`       | resolved from <stack>    | Run image reference
+| `<stack>`         | `CNB_STACK_PATH`      | `/cnb/stack.toml`        | Path to stack file (see [`stack.toml`](#stacktoml-toml))
+| `<tag>...`        |                       |                          | Additional tag to apply to exported image
+| `<uid>`           | `CNB_USER_ID`         |                          | UID of the stack `User`
+
+- **If** `<daemon>` is `false`, `<image>` MUST be a valid image reference
+- **If** `<daemon>` is `true`, `<image>` MUST be either a valid image reference or an imageID
+- **If** `<run-image>` is not provided by the platform the value will be resolved from the contents of stack
+- The lifecycle MUST accept valid references to non-existent images without error.
+- The lifecycle MUST ensure that the `<run-image>` is compatible with the `<previous-image>`. To be compatible, the following conditions must be met:
+  - Stack IDs must match
+  - Platform/Architecture must be the same
+  - `<run-image>` `mixins` must be a superset of `<previous-image>` mixins
+- The lifecycle MUST ensure registry read/write access to `<run-image>`, `<previous-image>`, `<tag>`
+
+##### Outputs
+| Output             | Description
+|--------------------|----------------------------------------------
+| [exit status]      | (see Exit Code table below for values)
+| `/dev/stdout`      | Logs (info)
+| `/dev/stderr`      | Logs (warnings, errors)
+| `<analyzed>`       | Analysis metadata (see [`analyzed.toml`](#analyzedtoml-toml)
+
+| Exit Code | Result|
+|-----------|-------|
+| `0`       | Success
+| `11`      | Platform API incompatibility error
+| `12`      | Buildpack API incompatibility error
+| `1-10`, `13-99` | Generic lifecycle errors
+| `30-39` | Analysis-specific lifecycle errors
+
+- The lifecycle MUST write [analysis metadata](#analyzedtoml-toml) to `<analyzed>`.
```

Comment:
```suggestion

- The lifecycle MUST write [analysis metadata](#analyzedtoml-toml) to `<analyzed>`, where:
  -  `build-image` MUST match the values provided in `<stack>`
  -  `run-image` MUST describe the `<run-image>`
  - **If** the `<previous-image>` is compatible with the `<run-image>` `previous-image` MUST describe `<previous-image>`.  `<previous-image>` is compatible with `<run-image>` if:
    - The stack IDs match
    - `<run-image>` `mixins` are a superset of `<previous-image>` mixins
  - **Else** the lifecycle MUST omit `previous-image.reference` and `previous-image.metadata`
```

Reply:
Added in 6480d2e975ba81d5b61d681f0177539408c834b8